onnx/shape_inference/implementation.h

```diff
@@ -77,22 +80,26 @@ struct GraphInferenceContext {
       const std::unordered_map<std::string, TypeProto*>&
           outer_scope_value_types_by_name_in,
       const std::unordered_map<std::string, int> opset_imports_in,
-      SymbolTable* symbolTable_in,
+      SymbolTable* symbol_table_in = nullptr,
       const ISchemaRegistry* schema_registry_in = OpSchemaRegistry::Instance(),
-      const int ir_version_in = IR_VERSION)
+      const int ir_version_in = IR_VERSION,
+      const ModelLocalFunctionsMap& model_local_functions_in = {},
+      GetFunctionMapIdFn get_func_id = nullptr)
```

Comment:
Do we really need this (function to combine the domain and opname into one string) as a parameter? I understand having a single place where we can change it easily, if we want to, but unclear if we need to expose it as a parameter (that others start taking a dependence on).

Reply:
+1 We should carefully review it since it's an important change that involves many functions and people will start to rely on it. It looks useful to me though. It will be more convincing if there are some real use cases for it.